docker-compose.yml

```diff
@@ -49,6 +54,7 @@ compile:
   command: env REBAR=rebar3 make compile
   image: logplex_base
   volumes:
+    - ./_build:/usr/src/app/_build
```

Comment:
Right, the implicit requirement for this to work, then, is to not do local compile at all (else, `rm -rf ./_build` before proceeding) ... but I wonder how we can do better. Can rebar be configured to use a different directory other than _build?